docs/development/malware-checks.rst

```diff
@@ -0,0 +1,109 @@
+Malware Checks
+==============
+
+.. contents::
+    :local:
+
+Background
+----------
+
+In September 2019, the Python Software Foundation issued a `Request for Proposal`_ for a system to automate the detection of malicious uploads. This system was initially rolled out in February 2020 by `pull request 7400`_.
+
+Overview
+------------
+This is a high-level diagram of the automated malware check system.
+
+|VerdictLifecycle|
+
+Checks can be triggered in the following ways:
+
+* A PyPI user uploads a new File, Release or Project;
+* A schedule;
+* A PyPI administrator initiates an evaluation run.
+
+All of the above triggers call the `IMalwareCheckService`_ factory to determine how to execute the check. On production, 
+the `DatabaseMalwareCheckService`_ is returned, which runs the check and produces one or more verdicts. PyPI administrator and moderators are continuously reviewing the verdicts in the warehouse admin, making determinations about the accuracy of checks, about taking further action if needed (e.g. removing a malicious package surfaced by a verdict).
```

Comment:
Grammar nit: "PyPI administrators and moderators continuously review ..." to avoid the gerund maybe?